### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by jakobbraun**

### CODE REVIEW

File: .github/workflows/ci-build-next-java.yml [hunk 1]
```diff
@@ -35,3 +35,4 @@ jobs:
         if: ${{ always() && github.event.pull_request.head.repo.full_name == github.repository && github.actor != 'dependabot[bot]' }}
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          fail_if_no_tests: false
```
Comment (morazow) [inline .github/workflows/ci-build-next-java.yml:38]: :+1:

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -22,9 +26,31 @@ jobs:
...
-        run: mvn -B clean verify sonar:sonar --file pom.xml -Dsonar.organization=exasol -Dsonar.host.url=https://sonarcloud.io -Dsonar.login=$SONAR_TOKEN
+        run: |
+          mvn --batch-mode clean verify \
+              -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
+              -DtrimStackTrace=false
+      - name: Publish Test Report
+        uses: scacap/action-surefire-report@v1
+        if: ${{ always() && github.event.pull_request.head.repo.full_name == github.repository && github.actor != 'dependabot[bot]' }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (morazow) [inline .github/workflows/ci-build.yml:44]: Should above condition be added here also?

**Review by morazow — APPROVED** [review]
Added minor question, but looks good!

File: .github/workflows/ci-build.yml (hunk 2 above)
Comment (jakobbraun) [reply .github/workflows/ci-build.yml:44]: Don't know. It's just the template update by project-keeper.

**Pull request merged**